programmers/크레인.js

```diff
@@ -0,0 +1,34 @@
+function solution(board, moves) {
+    
+    let result = [];
+    let temp = 0;
+    let disappear = 0;
+    
+    for (let i = 0; i < moves.length; i++) {
+        
+        for (let j = 0; j < board.length; j++) {
+            
+            if (board[j][moves[i] - 1] === 0) continue;      // 빈칸은 넘어가도록 하는 코드
+            
+            if (board[j][moves[i] - 1] > 0) {               // result라는 배열에 인형을 하나씩 담는 코드
+                result.push(board[j][moves[i] - 1]);
+                board[j][moves[i] - 1] = 0;
+                break;
+            }
+        }
+        let len = result.length;                    
```

Comment:
```suggestion
        let storageLength = result.length;                    
```
코테에는 상관없는데 개발들어갈때는 이렇게 확실한 변수명이 좋아요